pandas/tests/tools/test_tile.py

```diff
@@ -297,6 +297,10 @@ def test_single_bin(self):
         result = cut(s, 1, labels=False)
         tm.assert_series_equal(result, expected)
 
+        s = Series([0., 0.])
```

Comment:
This PR is related to that test. The test checks that pd.cut works fine when the input array contains identical values and bins=1. That is the general scenario, while this PR covers the specific case of an array contains all 0s. it would be misleading to move the test to another function.